crates/sui-graphql-rpc/src/types/transaction_block_effects.rs

```diff
@@ -197,6 +205,29 @@ impl TransactionBlockEffects {
         }
     }
 
+    async fn abort_code(&self, ctx: &Context<'_>) -> Result<Option<BigInt>> {
+        let resolver: &PackageResolver = ctx.data_unchecked();
+        let status = self.resolve_native_status_impl(resolver).await?;
+        match status {
+            NativeExecutionStatus::Success => Ok(None),
+            NativeExecutionStatus::Failure { error, .. } => match error {
+                ExecutionFailureStatus::MoveAbort(loc, code) => {
+                    Ok(Some(BigInt::from_str(&match resolver
+                        .resolve_clever_error(loc.module.clone(), code)
+                        .await
+                    {
+                        Some(CleverError {
+                            error_code: Some(clever_error_code),
+                            ..
+                        }) => clever_error_code.to_string(),
+                        _ => code.to_string(),
+                    })?))
```

Comment:
Instead of doing a `from_str` here, I'd probably just cast the `u8` from the `clever_error_code` to a `u64` and that should work I think and it's no longer fallible. That failing though, I'd push this into the different branches as I think that will be a bit nicer and more direct.

So maybe something like
```suggestion
                    Ok(Some(BigInt::from(&match resolver
                        .resolve_clever_error(loc.module.clone(), code)
                        .await
                    {
                        Some(CleverError {
                            error_code: Some(clever_error_code),
                            ..
                        }) => clever_error_code as u64,
                        _ => code,
                    })))
```